uta_tools/uta_tools.py

```diff
@@ -569,3 +570,77 @@ def _get_exon_number(tx_exons: List, tx_pos: int) -> int:
                 break
             i += 1
         return i
+
+    def _get_fasta_aliases(self, sequence_id: str, prefixes: List[str]) -> List[str]:
+        """Retrieve equivalent aliases for FASTA headers
+        :param sequence_id: ID to look up aliases for
+        :param prefixes: ordered list of requested namespaces, case-sensitive
+        :return: List (possibly empty) of matching aliases
+        :raise: KeyError if SeqRepo raises a translation error
+        """
+        other_ids, warnings = self.seqrepo_access.translate_identifier(sequence_id)
+        if warnings:
+            raise KeyError(f"Unable to find other IDs for {sequence_id}")
+        aliases = []
+        for prefix in prefixes:
+            for other_id in other_ids:
+                if other_id and other_id.startswith(prefix):
+                    aliases.append(other_id)
+        return aliases
```

Comment:
I believe this does the same thing
```suggestion
        aliases, warnings = self.seqrepo_access.translate_identifier(sequence_id,
                                                                     prefixes)
        if warnings:
            raise KeyError(f"Unable to find other IDs for {sequence_id}")
        return aliases
```